lib/commands/general.js

```diff
@@ -13,24 +13,28 @@ commands.getWindowSize = async function getWindowSize () {
   }
   // workaround for https://github.com/microsoft/WinAppDriver/issues/1104
   log.info('Cannot retrieve window size from WinAppDriver. ' +
-    'Falling back to WMIC usage');
-  const {stdout} = await exec('wmic.exe', [
-    'path', 'Win32_VideoController',
-    'get', 'VideoModeDescription,CurrentVerticalResolution,CurrentHorizontalResolution',
-    '/format:value',
-  ]);
-  log.debug(stdout);
-  const widthMatch = /CurrentHorizontalResolution=(\d+)/.exec(stdout);
-  const heightMatch = /CurrentVerticalResolution=(\d+)/.exec(stdout);
-  if (!widthMatch || !heightMatch) {
+      'Falling back to Windows Forms to calculate dimensions');
+  const stdout = await getDimensions();
```

Comment:
you could parse stdout directly in getDimensions call and return the dictionary from there

Reply:
Thanks!  Implemented this :)